### PR TITLE
chore: add retry on build pipeline

### PR DESCRIPTION
**Pull request by purelind**
Add retry.

### CODE REVIEW

**Review by lijie0123 — APPROVED** [review]

File: jenkins/pipelines/cd/atom-jobs/docker-common.groovy [hunk 1]
```diff
@@ -64,7 +64,9 @@ if (params.PRODUCT.length() <= 1) {
 binarys = params.INPUT_BINARYS.split(",")
 def download() {
     for (item in binarys) {
-        sh "curl ${FILE_SERVER_URL}/download/${item} | tar xz"
+        retry(3) { 
+            sh "curl ${FILE_SERVER_URL}/download/${item} | tar xz"
```
Comment (wuhuizuo) [inline jenkins/pipelines/cd/atom-jobs/docker-common.groovy:68]: should add fail option

**Pull request merged**